setup.py

```diff
@@ -67,7 +67,7 @@
             'flake8-quotes',
             'codecov',
             'tchannel>=0.27', # This is only used in python 2
-            'opentracing_instrumentation>=2,<3',
+            'opentracing_instrumentation',
```

Comment:
I prefer to keep the ranges, e.g. previously`>=1,<2` was not compatible with new opentracing-2.x

Reply:
Done.